test-packages/e2e-tests/test/on-prem/odata.spec.ts

```diff
@@ -54,3 +76,13 @@ describe('OData OnPrem', () => {
       });
   }, 60000);
 });
+
+describe('OData Cloud', () => {
```

Comment:
[pp] perhaps move this test to a file not in the `on-prem` folder?

Reply:
I renamed the directory to `proxy`.